src/components/template-base.tsx

```diff
@@ -31,18 +32,20 @@ const ThemeContext = React.createContext(getThemeConfig());
 export const useTheme = () => React.useContext(ThemeContext);
 
 export interface TemplateBaseProps extends React.PropsWithChildren<{}> {
+  subject?: string;
   isExtruded?: boolean;
   language?: string;
 }
 
 export const TemplateBase = (props: TemplateBaseProps) => {
-  const { isExtruded = false, language } = props;
+  const { isExtruded = false, language, subject } = props;
```

Comment:
Should solve type error with this?
```suggestion
  const { isExtruded = false, language, subject = '' } = props;
```